pkg/provisioner/controller.go

```diff
@@ -464,21 +513,24 @@ func (c *Controller) deleteResources(ob *v1alpha1.ObjectBucket, cm *corev1.Confi
 }
 
 // Add a finalizer to the OBC so that resource deletion can be orchestrated.
-func (c *Controller) protectOBC(obc *v1alpha1.ObjectBucketClaim) (err error) {
+// Add custom labels to the OBC.
+func (c *Controller) setOBCMetaFields(obc *v1alpha1.ObjectBucketClaim) (err error) {
 
 	clib := c.libClientset
-	obcNsName := obc.Namespace + "/" + obc.Name
 
+	logD.Info("getting OBC to set metadata fields")
 	obc, err = clib.ObjectbucketV1alpha1().ObjectBucketClaims(obc.Namespace).Get(obc.Name, metav1.GetOptions{})
 	if err != nil {
-		return fmt.Errorf("unable to Get obc %q for protection: %v", obcNsName, err)
+		return fmt.Errorf("error getting unconfigured obc: %v", err)
```

Comment:
what does "unconfigured" mean? May be better to omit?

Reply:
fair point - I was struggling to think of how to describe these yet-unclaimed OBCs.